src/CMakeLists.txt

```diff
@@ -8,27 +8,53 @@
 
 include_directories(.)
 
+# Added as part of the merge with build only infrastructure:
 add_subdirectory(common)
+
+if(NOT INFRASTRUCTURE_ONLY)
```

Comment:
Where does the INFRASTRUCTRUE_ONLY option come from?

Reply:
> Where does the INFRASTRUCTRUE_ONLY option come from?

Hi David -- Taking a step back to explain the "big picture", remember that we are using RAJAPerf Suite (RPS) in two distinct manners:  1) Provide Kokkos translations of RPS kernels that can be run along side native RPS kernels, and 2) Using RPS as the **_infrastructure driver_** for Kokkos Kernels and Kokkos performance tests that we have written.  To be clear, this second case has nothing to do with RPS kernels & their Kokkos translations.   This second case is tied to the "INFRASTRUCTURE_ONLY" option.  Does that make sense?

Reply:
The _why_ makes sense, I meant that INFRASTRUCTURE_ONLY should be explicitly defined as a CMake `option` (https://cmake.org/cmake/help/latest/command/option.html)  with a default value of False, that you can override when you are using the perfsuite as the infrastructure driver.